en/manual/index.md

```diff
@@ -5,15 +5,15 @@
 These pages contain information about how to use Stride, an open-source C# game engine.
 
 > [!Note]
-> The Stride manual is under construction and updated regularly with new content. Follow [Stride on Twitter](https://twitter.com/stridedotnet?lang=en) for documentation updates.
+> The Stride manual is under construction and updated regularly with new content. Follow [Stride on X](https://x.com/stridedotnet?s=20) for documentation updates.
 
 ## Latest documentation
 
 ### Recent updates
 - <span class="badge text-bg-info">Updated</span> [Linux - Setup and requirements](platforms/linux/setup-and-requirements.md) - Fedora OS option added
 - <span class="badge text-bg-success">New</span> [Scripts - Serialization](scripts/serialization.md) - Serialization explained
 - <span class="badge text-bg-info">Updated</span> [Scripts - Public properties and fields](scripts/public-properties-and-fields.md) - Content improvements and additions
-- <span class="badge text-bg-success">New</span> [Engine - Entity Component model - Usage](engine/entity-component-model/usage.md) - ECS usage explained
+- <span class="badge text-bg-success">New</span> [Engine - Entity Component model - Usage](engine/entity-component-system/usage.md) - ECS usage explained
 - <span class="badge text-bg-info">Updated</span> [Engine - Entity Component model](engine/entity-component-model/index.md) - Content improvements
```

Comment:
Here is the last escapee to be changed 🤣

Reply:
..Line 17

Reply:
Now, I'm truly done @VaclavElias

Reply:
> ..Line 17

Done and done! :)